controllers/mysqlcluster_controller.go

```diff
@@ -871,8 +871,14 @@ func (r *MySQLClusterReconciler) reconcileV1BackupJob(ctx context.Context, req c
 			VolumeSource: *jc.WorkVolume.DeepCopy(),
 		}}
 
+		var bucketConfig mocov1beta2.BucketConfig
+
+		if err := mocov1beta1.Convert__BucketConfig_To_v1beta2_BucketConfig(&jc.BucketConfig, &bucketConfig, nil); err != nil {
+			return fmt.Errorf("failed to convert bucket config from v1beta1 to v1beta2: %w", err)
+		}
```

Comment:
We should have retrieved BackupPolicy with v1beta2 at line 841 so that
this conversion can be removed.
https://github.com/cybozu-go/moco/pull/350/commits/4f5c0ae1595d2b591397a17b7fd8ac68133421cc#diff-67f34c9b4cabb4950b383eab55b6ebeeebc524fa70c888bdc132662c16a7899fR841

Reply:
I assumed that I would only create the v1beta2 API for `MySQLCluster` and continue to use v1beta1 without upgrading the `BackupPolicy`.
This conversion is necessary because `MySQLCluster` (v1beta2 package) and `BackupPolicy` (v1beta1 package) use a common `JobConfig` structure.

Or should I create a v1beta2 API for BackupPolicy too, even if there are no changes to the API?

Reply:
@d-kuro 
> Or should I create a v1beta2 API for BackupPolicy too, even if there are no changes to the API?

I think so.

@zoetrope Thoughts?

Reply:
I think the same version would be better.

Reply:
I added `v1beta2.BackupPolicy`

* Add BackupPolicy v1beta2: https://github.com/cybozu-go/moco/pull/350/commits/d0fa6af7719539826eacb0b46bcb87714462534b
* Use BackupPolicy v1beta2: https://github.com/cybozu-go/moco/pull/350/commits/7c02683f30de1d502d84cdd40fb4b51d1616fe8c
* Add conversion webhook for BackupPolicy: https://github.com/cybozu-go/moco/pull/350/commits/4cc80be617fe401251a76ab0f1e9c9f1b7d9985d